src/interfaces.ts

```diff
@@ -199,6 +199,13 @@ export interface CompilerPlugin {
     afterFileTranspile?: (entry: TranspileObj) => void;
     beforeFileDispose?: (file: BscFile) => void;
     afterFileDispose?: (file: BscFile) => void;
+
```

Comment:
Events are sorted by target, program events, scope events, file events

Reply:
Ok. I added comments in the interface explaining that, and moved the events to the proper locations.